qcodes/instrument_drivers/Keysight/Keysight_N5173B.py

```diff
@@ -1,5 +1,11 @@
 from qcodes.instrument_drivers.Keysight.Keysight_N5183B import N5183B
```

Comment:
this should use the new name, right?
```suggestion
from qcodes.instrument_drivers.Keysight.Keysight_N5183B import KeysightN5183B
```

Reply:
That would indeed be more forward looking